spec/requests/api_stories/move_workflows/court_to_prison/person_with_nomis_prison_number_spec.rb

```diff
@@ -0,0 +1,181 @@
+# frozen_string_literal: true
+
+require 'rack/test'
+
+# GIVEN a person with a prison number or police national computer number
+# WHEN the supplier wants to move the person from court to prison
+# THEN the supplier should call the api as outlined in the example below
+
+RSpec.describe 'court to prison move', type: :request do
+  include Rack::Test::Methods
+  include_context 'with mock prison-api'
+  include_context 'with Nomis alerts reference data'
+
+  let(:prison_number) { 'G8133UA' }
+  let(:prison) { create(:location, :prison) }
+  let(:court) { create(:location, :court) }
+
+  # rubocop:disable Rails/HttpPositionalArguments
+  let(:get_people_by_prison_number) do
```

Comment:
Feels really weird using memoized anonymous functions/lets to make requests to our api 🤔 .

Reply:
I guess it's cleaner than making the calls in the example body and equivalent to having methods with member variables so not sure if I'm suggesting anything here.

Reply:
🤷  personally I like the clean high-level example (`get_foo`, `post_bar` etc), but I'm interested in a consensus view on this